src/library/basesqltablemodel.cpp

```diff
@@ -808,13 +808,20 @@ Qt::ItemFlags BaseSqlTableModel::readOnlyFlags(const QModelIndex &index) const {
     return defaultFlags;
 }
 
-const QLinkedList<int> BaseSqlTableModel::getTrackRows(TrackId trackId) const {
-    QHash<TrackId, QLinkedList<int> >::const_iterator it =
+QVector<int> BaseSqlTableModel::getTrackRows(TrackId trackId) const {
```

Comment:
From an efficiency perspective, is this better?

The linked list return is a copy-on-write clone of the value, while this allocates a new qvector and copies its contents manually, while doing two hash table lookups (one for the count, one for the loop).